releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -135,7 +135,15 @@
 		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/master/lastStableBuild/artifact/build/p2-repository/"/>
 	</location>
 	
-	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
i am not sure if i like the tp to be changed and to be used for both the tests and building the update site

Reply:
Then we should discuss together with @kthoms. We have worked together to get rid of the extra target definition. Intention was to have the tests running against the same target as the user probably has when using the update site. Does not look to bad an idea for me ...